src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -196,6 +196,11 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilter
   display: flex;
   background-color: $BCgovBlue5O;
   padding: 1.25rem;
+
+  .people-roles-title {
+    padding-left: .5rem;
```

Comment:
Yes, this looks good, but I compared this label with the one for "Your Company" and that one has some top padding that doesn't look good. Could you please remove the top padding for class "define-company-class"?

Reply:
Fixed